docs/release_notes/release_notes-0.16.0.rst

```diff
@@ -59,6 +59,7 @@ Key Features and Updates
   * FIX-#5042: Fix series __getitem__ with invalid strings (#5048)
   * FIX-#4691: Fix binary operations between virtual partitions (#5049)  
   * FIX-#5045: Fix ray virtual_partition.wait with duplicate object refs (#5058)
+  * FIX-#4016, FIX-#4086, FIX-#4039: Fall back to pandas in case of duplicate column names (#4896)
```

Comment:
```suggestion
```
We already hit 0.16.0 release.

Reply:
Do you mean I have to create a new file docs/release_notes/release_notes-0.17.0.rst and add this line to the new file?

Reply:
There is no need for that, please remove. Since 0.16.0 release we plan to automate release note generation going forward. See #5095 and #4747 for more info.

Reply:
Done.